test/src/dftbp/unit/wavegrid/generate_ylm_tests.py

```diff
@@ -0,0 +1,105 @@
+""" Calculate reference values using scipy to check the Fortran spharmonics implementation """
```

Comment:
Does this file belong in the unit test part of the source tree, or is the utils/ part of the repoitory more appropriate?